pkg/shell/shell.less

```diff
@@ -423,15 +423,17 @@ html.index-page body {
                 }
 
                 a {
-                    padding: 0.5rem;
+                    padding: 0.5rem 1rem;
                     display: block;
                     background: var(--color-ct-list-bg);
                     color: var(--color-ct-list-text);
+                    text-decoration: none;
 
                     > span {
                         display: block;
-                        text-indent: ~"calc(1ex - 18px)";
-                        padding-left: ~"calc(1ex + 18px)";
+                        text-overflow: ellipsis;
```

Comment:
This truncation of hostnames unrelated to #12210 and I would rather see this commit made as a separate PR.